drivers/volume/portworx/portworx.go

```diff
@@ -100,6 +100,7 @@ const (
 	pxReleaseManifestURLEnvVarName            = "PX_RELEASE_MANIFEST_URL"
 	pxServiceLocalEndpoint                    = "portworx-service.kube-system.svc.cluster.local"
 	mountGrepVolume                           = "mount | grep %s"
+	deviceMapper                              = "mount | grep %s | awk '{print $1}'"
```

Comment:
[nit] name this `mountGrepSecondColumn` or something: `deviceMapper` isn't very informative by itself

Reply:
Done.